modules/ocf_kvm/facts.d/vms

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+
+
+def run_command(command_args):
+    output = subprocess.check_output(command_args).decode('utf-8').split('\n')
```

Comment:
there's a `str.splitlines()` you could use

Reply:
Ah, nice, that's a bit better.